### DISCUSSION TITLE
why not read aloud (TTS) website.

### DISCUSSION THREAD

**Discussion by trionline1234** [immersive-translate/immersive-translate#1101] [Ideas]
### 新功能 | New Feature

 Read aloud website (Text to speech) with bilingual language.

With each sentence, first , extension will read aloud with mother language, after that, extension will read aloud with original language of text many times with different speeds.

### 为什么您想要添加这些功能 | Why you want to add these features

It will help for foreign language learner can practice listening with bilingual audio.
Todays, most extension with TTS only read text with just one language (such as https://chrome.google.com/webstore/detail/read-aloud-a-text-to-spee/hdhinadidafjejdhmfkjgnolgimiaplp)

**Comment by KnIfER** [comment]
一键翻译，一键朗读：


https://github.com/user-attachments/assets/fe9783d8-cef7-403c-b527-659fa33f2833

**Comment by theowenyoung** [comment]
I'm sorry, but this is not a direction we want to consider in the short term.
